apps/internal-tool/.env.development

```
@@ -0,0 +1,6 @@
+NEXT_PUBLIC_STACK_API_URL=http://localhost:${NEXT_PUBLIC_STACK_PORT_PREFIX:-81}02
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The NEXT_PUBLIC_STACK_API_URL key is not assigned properly

(SubstitutionKey)